adagrams/game.py

```diff
@@ -1,11 +1,184 @@
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    import string
+    import random
+    import copy
+
+    user_hand = []
+    # deepcopy makes a copy that has its own reference in memory so that we don't
+    # overwrite the original list
+    temp_dict = copy.deepcopy(LETTER_POOL)
+
+    while len(user_hand) < 10:
+        # random.choice selects a random letter from the string we give it and then
+        # string.ascii_uppercase gives us a string of all 26 letters in upper case form
+        letter = random.choice(string.ascii_uppercase)
+        letter_count = temp_dict[letter]
+        if letter_count == 0:
```

Comment:
If we renamed temp_dict to be more descriptive of its contents, we could combine these lines without losing readbility.
```suggestion
        if letter_counts[letter] == 0:
```